### PR TITLE
Finalized the mobile-version Portfolio

### PR DESCRIPTION
**Pull request by ehmaddd**
### Finalized the mobile-version Portfolio

This portfolio is mobile-first and it will not only work fine on a desktop computer but also on mobiles because it is responsive and adjusts itself according to the screen. I have used CSS flexbox and grid in it and by the culmination of both, I've created it.

Regards

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,9 +11,10 @@
     <title>Portfolio</title>
 </head>
 <body>
-    <section class="flex-container outer-container">
-        <header class="flex-container">
-            <nav class="flex-container nav-bar">
+    <!--- BRIEF INTRO -->
+    <section class="flex-container outer-container top-portion">
+        <header class="flex-container top-portion">
+            <nav class="flex-container top-portion nav-bar">
```
Comment (joel-kalema) [inline index.html:17]: - [ ] You did great work @ehmaddd 😃 , kindly make sure to stick to the design as much as possible (`font`, `colors`, `images`, `text`, `margins`, `background image`) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.)  in this milestone🤗🤗


   
> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |heade| | header| |

File: index.html [hunk 2]
```diff
@@ -46,9 +47,259 @@
...
+                                    <ul>
+                                        <li>Ruby</li>
+                                        <li>HTML</li>
+                                        <li>CSS</li>
+                                        <li>PHP</li>
+                                    </ul>
+                                </div>
+                                <div class="see-project"><button class="see-project-btn">See Project</button></div>
+                            </div>
+                        </div>
```
Comment (joel-kalema) [inline index.html]: - [ ] Good job @ehmaddd 😃, but you need to fix same part  such as (`margin`, and `padding`)

     - Kindly, make sure you fixed the margin of your `work-item` to stick to the given [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988&t=DlZ5PelwOdZvAKL2-0) 👍 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |projecto| | project| |

- Kindly, make sure you fixed the padding your languages used to stick to the given [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988&t=DlZ5PelwOdZvAKL2-0) 👍  

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |langagesog| |langages| |

File: index.html [hunk 3]
```diff
@@ -46,9 +47,259 @@
...
+                <div class="about-me-heading"><h1>About Me</h1></div>
+            </div>
+
+            <div class="grid-item me-detail-grid">
+                <div class="detail-para"><p>I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p></div>
+            </div>
+
+            <div class="grid-item see-btn-grid">
+                <button class="see-project-btn">Get my Resume</button>
+            </div>
```
Comment (joel-kalema) [inline index.html]: - [ ]   Good job @ehmaddd 😃, but you need to fix some parts such as (`the margin`, `font size`, `and color`) and you need to fix your background image to stick to the given [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988&t=DlZ5PelwOdZvAKL2-0) 👍
> The h1 color: #172B4D; font-weight: 700; font-size: 40px;
> The p color: #42526E; font-weight: 400; font-size: 20px;

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |aboutfg| |about| |

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @ehmaddd 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 

- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ehmaddd) [reply index.html:17]: I've applied this change. and now ready to re-submit it

File: index.html (hunk 2 above)
Comment (ehmaddd) [reply index.html]: Yes. This one also changed with replacing h3 with h5

File: index.html (hunk 3 above)
Comment (ehmaddd) [reply index.html]: Also fixed this according to the requirement

**Review by jerryowusu — APPROVED** [review]
Hi @ehmaddd , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

200w


## Highlights
No linter errors :heavy_check_mark: 
Great job so far :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**